chunks/lang-ruby/README.md

```diff
@@ -0,0 +1,52 @@
+# workspace-ruby-*
+
+## How to use for a Gitpod workspace
+
+Put the following line in your `.gitpod.yml`:
+
+If you do not have a `.gitpod.yml`, run `gp init` on your terminal to create one.
+
+```yaml
+image: gitpod/workspace-ruby-3.1
+```
+
+Lastly, [see it in action!](https://www.gitpod.io/docs/introduction/learn-gitpod/gitpod-yaml#see-it-in-action)
+
+### Extra docker image variants
+
+We provide multiple variants of the `gitpod/workspace-ruby-*` docker image as per [chunk.yaml](./chunk.yaml). Each containing a different version of Ruby:
+
+- `gitpod/workspace-ruby-2.7` (This is ruby 2.7.6)
+- `gitpod/workspace-ruby-3.0` (This is ruby 3.0.4)
+- `gitpod/workspace-ruby-3.1` (This is ruby 3.1.2)
```

Comment:
This is will rapidly get out of date. Could we change to the following:
```suggestion
- `gitpod/workspace-ruby-2.7` (This being ruby 2.7.x)
- `gitpod/workspace-ruby-3.0` (This being ruby 3.0.x)
- `gitpod/workspace-ruby-3.1` (This being ruby 3.1.x)
```

It applies also to other images of different languages.

Reply:
Thanks for pointing this out, will do.